### PR TITLE
Rescue exceptions caused by FetchLinkCardService

### PR DESCRIPTION
**Pull request by abcang**
Rescue exceptions caused by FetchLinkCardService

### CODE REVIEW

File: app/models/concerns/remotable.rb [hunk 1]
```diff
@@ -10,7 +10,11 @@ module Remotable
       method_name = "#{attribute_name}=".to_sym
 
       define_method method_name do |url|
-        parsed_url = Addressable::URI.parse(url).normalize
+        begin
+          parsed_url = Addressable::URI.parse(url).normalize
```
Comment (abcang) [inline app/models/concerns/remotable.rb]: `Addressable::URI::InvalidURIError` is raised when executing `Addressable::URI.parse` with an illegal URL as an argument.

File: app/models/concerns/remotable.rb [hunk 2]
```diff
@@ -26,7 +30,7 @@ module Remotable
           send("#{attachment_name}_file_name=", filename)
 
           self[attribute_name] = url if has_attribute?(attribute_name)
-        rescue HTTP::TimeoutError, OpenSSL::SSL::SSLError, Paperclip::Errors::NotIdentifiedByImageMagickError, Addressable::URI::InvalidURIError => e
+        rescue HTTP::TimeoutError, HTTP::ConnectionError, OpenSSL::SSL::SSLError, Paperclip::Errors::NotIdentifiedByImageMagickError, Addressable::URI::InvalidURIError => e
```
Comment (abcang) [inline app/models/concerns/remotable.rb]: `HTTP::ConnectionError` is raised when the connection fails with `http_client.get(url)`.

File: app/services/fetch_link_card_service.rb [hunk 3]
```diff
@@ -18,6 +18,8 @@ def call(status)
     return if res.code != 200 || res.mime_type != 'text/html'
 
     attempt_opengraph(card, url) unless attempt_oembed(card, url)
+  rescue HTTP::ConnectionError, OpenSSL::SSL::SSLError
```
Comment (abcang) [inline app/services/fetch_link_card_service.rb]: The exception is raised when the connection fails or the HTTPS connection fails with `http_client.head (url)`.

**Review by clworld — APPROVED** [review]
Looks good to me.

**Pull request merged**